probe.c

```diff
@@ -159,11 +164,10 @@ void probe_move_to_sensor(float * target, float feed_rate, uint8_t invert_feed_r
   probe.isprobing = 1;
 
   sys.state = STATE_PROBING;
-  
-  if (!probe_loop())
-    return;
-
+ 
   uint8_t probe_fail;
+  probe_fail = !probe_loop();
```

Comment:
Only if sensor == MAG_SENSOR. There was an existing check for probe_fail in the case of the MAG_SENSOR which depends on the PROBE_ACTIVE macro, which I kept.